Packs/Illumio/Integrations/IllumioCore/IllumioCore_test.py

```diff
@@ -22,6 +25,7 @@
 INVALID_VISIBILITY_LEVEL_EXCEPTION_MESSAGE = "{} is an invalid visibility level."
 INVALID_BOOLEAN_EXCEPTION_MESSAGE = "Argument does not contain a valid boolean-like value"
 INVALID_MAX_RESULTS_EXCEPTION_MESSAGE = "{} is an invalid value for max results. Max results must be between 0 and 500"
+INVALID_VALUE_EXCEPTION_MESSAGE = "{} is an invalid for {}."
```

Comment:
Shouldn't this be `{} is an invalid value for {}`? This doesn't seem to be used anywhere though, so maybe just remove it.